pyproject.toml

```diff
@@ -33,31 +33,37 @@ classifiers = [
 ]
 dependencies = [
   # Core generation dependencies, pinned for reproducible builds.
-  "accelerate==0.30.1",
+  "accelerate==0.33.0",
+  "bitsandbytes==0.43.3",
   "clip_anytorch==2.6.0",       # replacing "clip @ https://github.com/openai/CLIP/archive/eaa22acb90a5876642d0507623e859909230a52d.zip",
   "compel==2.0.2",
   "controlnet-aux==0.0.7",
-  "diffusers[torch]==0.27.2",
+  # TODO(ryand): Bump this once the next diffusers release is ready.
+  "diffusers[torch] @ git+https://github.com/huggingface/diffusers.git@4c6152c2fb0ade468aadb417102605a07a8635d3",
```

Comment:
- Do we need a different version of `diffusers` for FLUX?
- Does this conflict with #6740? 
- Are we introducing issues with single-file loading, as described in #6740, with whatever version is needed by this PR?

Reply:
We shouldn't need to make any changes to the diffusers version anymore now that we're no longer using it in flux

Reply:
I take back what I said, quanto seems to require a more recent version of diffusers. We may need to just wait for that diffusers PR to get merged

Reply:
quanto being one of the modules we use for managing quantized models

Reply:
We've dropped quanto and are only using bitsandbytes for quntization for now. I've downgraded diffusers back to its original version.